recipes/openexr/config.yml

```diff
@@ -1,3 +1,3 @@
 versions:
   "2.3.0":
-    folder: all
+    folder: "2.3.0"
```

Comment:
Why moving from all to a version folder?

Reply:
Because of 2.4.0: https://github.com/conan-io/conan-center-index/pull/449

Reply:
ok, thanks!